src/runmode-af-packet.c

```diff
@@ -287,8 +287,16 @@ static void *ParseAFPConfig(const char *iface)
     }
 
     if (ConfGetChildValueWithDefault(if_root, if_default, "copy-iface", &out_iface) == 1) {
-        if (strlen(out_iface) > 0) {
-            aconf->out_iface = out_iface;
+        if (out_iface != NULL) {
```

Comment:
think this isn't needed as it wasn't needed before?

Reply:
The following config segfaults on the strlen call if I don't do this
```
 - interface: eth2        
    threads: auto          
    cluster-id: 98         
    cluster-type: cluster_flow
    defrag: yes            
    copy-mode: ips         
    copy-iface: 
```
Should this be ignored?

Reply:
Nope, good that you tested it. Wonder if we use this pattern in more places.

Reply:
I think it is easy to miss indeed so there may be more places. I think I can come up with a cocci script to check it. Will do so.